src/main/java/ch/wisv/events/sales/controller/stats/SalesStatsMainController.java

```diff
@@ -0,0 +1,54 @@
+package ch.wisv.events.sales.controller.stats;
+
+import ch.wisv.events.core.model.customer.Customer;
+import ch.wisv.events.core.service.auth.AuthenticationService;
+import ch.wisv.events.sales.service.SalesService;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+/**
+ * SalesScanMainController class.
+ */
+@Controller
+@RequestMapping(value = "/sales/stats")
+@PreAuthorize("hasRole('USER')")
+public class SalesStatsMainController {
+
+    /**
+     * AuthenticationService.
+     */
+    private final AuthenticationService authenticationService;
+
+    /**
+     * SalesService.
+     */
+    private final SalesService salesService;
+
+    /**
+     * SalesStatsMainController.
+     *
+     * @param authenticationService of type AuthenticationService.
+     * @param salesService          of type SalesService.
+     */
+    public SalesStatsMainController(AuthenticationService authenticationService, SalesService salesService) {
+        this.authenticationService = authenticationService;
+        this.salesService = salesService;
+    }
+
+    /**
+     * Index view of the stats page.
+     *
+     * @param model of type Model.
+     * @return String
+     */
+    @GetMapping
+    public String index(Model model) {
+        Customer currentUser = authenticationService.getCurrentCustomer();
```

Comment:
You could alternatively add this as a method parameter and it will be autowired if it uses the Spring Security Context. This works too though

Reply:
I tried this, but I can only get the authentication. I could match the authentication with the customer in the function itself, but I think the current location of the getCurrentCustomer function is better placed in the authenticationService.